src/Services/MarketTradingDataService.cs

```diff
@@ -19,75 +19,127 @@ public MarketTradingDataService(ICandlesHistoryServiceProvider candlesHistorySer
             _candlesHistoryServiceProvider = candlesHistoryServiceProvider;
         }
         
-        public async Task<AssetPairTradingData> TryGetPairAsync(string assetPair)
+        public async Task<AssetPairTradingDataItem<double>> TryGetPairVolumeAsync(string assetPair)
         {
-            var spotCandlesTask = GetLastDayCandlesAsync(MarketType.Spot, assetPair);
-            var mtCandlesTask = GetLastDayCandlesAsync(MarketType.Mt, assetPair);
+            var candles = await GetLastDayCandlesAsync(assetPair); // Item1 = spot, Item2 = MT
 
-            var spotCandles = await spotCandlesTask;
-            var mtCandles = await mtCandlesTask;
+            var volume24H = candles.Item1.History.Sum(c => c.TradingOppositeVolume) +
+                            candles.Item2.History.Sum(c => c.TradingOppositeVolume);
 
-            var lastTradePrice = spotCandles.History.LastOrDefault()?.LastTradePrice
-                            ?? mtCandles.History.LastOrDefault()?.LastTradePrice
-                            ?? 0;
+            return new AssetPairTradingDataItem<double>(assetPair, volume24H);
+        }
+
+        public async Task<AssetPairTradingDataItem<double>> TryGetPairLastTradePriceAsync(string assetPair)
+        {
+            var candles = await GetLastDayCandlesAsync(assetPair); // Item1 = spot, Item2 = MT
```

Comment:
To calculate last trade price we don't need to request entire day by hours, any single candle will be enough